### ISSUE TITLE
HTTPS certificate expired

### ISSUE THREAD

**Issue by buccalon** [imaginerio/omeka-s-docker#16]
![image](https://user-images.githubusercontent.com/476699/137535464-cb2c071d-3145-44e2-9a41-9d5809e6bef6.png)

![image](https://user-images.githubusercontent.com/476699/137535522-031892fd-d9bf-4ae2-b744-b18d5009565f.png)

**Comment by davidheyman** [comment]
There was a problem with letsencrypt where it wouldn't renew our certificate because it said we had too many failed attempts. I waited for the counter to reset for 48 hours and still no luck so I set up a new domain at http://images2.imaginerio.org/ and generated a new certificate which worked fine.

I've moved our stuff to point to the new domain, but not sure what we want to do about Smapshot. Perhaps wait until the static manifests are up and deal with things then?

**Comment by martimpassos** [comment]
Yeah we should be good to go with static manifests by Friday. I'd wait till there and give them the new collection address
